### DISCUSSION TITLE
math in lists or tables

### DISCUSSION THREAD

**Discussion by nschloe** [community/community#16992] [Other Feature Feedback, Questions, & Ideas] [answered]
The recently released math support has some shortcoming.

- It doesn't work in lists $a+b$

It doesn't work in tables:

| $a+b$    |
| -------- |
| $c+d$    |

**Comment by diomidova** [comment]
It looks fine for me. Has this been fixed?

**Reply by cychitivav** [reply]
Sometimes it works for me but sometimes it does not. In the notes or quotes it still doesn't work for me

> __Note__: $a=1$

> __Warning__: $b=2$

**Reply by nschloe** [reply]
Yeah, other things aren't working either, e.g., https://github.com/github/feedback/discussions/17325.

**Comment by nschloe** [comment, marked as answer]
This is fixed now.
